test/verify/check-machines

```diff
@@ -2532,7 +2532,8 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
                 else:
                     raise AssertionError("Unknown disk device")
             else:
-                b.wait_in_text("#vm-{0}-disks .ct-table-empty".format(name), "No disks defined")
+                # On smaller resolutions it is not visible, but always in DOM
+                b.wait_present("#vm-{0}-disks .ct-table-empty:contains('No disks defined')".format(name))
```

Comment:
Hard to say without a screenshot, but could we make this a `# HACK` and conditional to firefox?

Reply:
Also, judging by the amount of hacks we had to do in selenium/edge tests (which does have that weird "scrolled off == !visible" behaviour), we should run into this in more cases if it was generally an issue.

Reply:
This is not that kind of hack, that it is visible, just the protocol does not think so. It literally gets hidden when you make your window smaller. (Just go to VM details that has no disks/networks and make your window smaller and bigger).

I changed it to test it being visible on chrome and not visible (but present) on firefox. This way we can check it works in both ways. I don't call it hack for that reason.